polars/dataframe.ts

```diff
@@ -1755,6 +1801,66 @@ export const _DataFrame = (_df: any): DataFrame => {
     set columns(names) {
       _df.columns = names;
     },
+    /**
+     * Return back text/html and application/vnd.dataresource+json representations of the DataFrame
+     * @returns Media bundle / mimetype keys for Jupyter frontends
+     */
+    [Symbol.for("Jupyter.display")]() {
```

Comment:
We could probably reuse some of this outside of just jupyter. Would you be willing to refactor the html rendering into a `df.toHTML()` function?

Reply:
Totally.

Reply:
How long before someone asks for a CSS option to this func? :)

Reply:
I hope at that point they realize how to use `toRecords` themselves.